web/src/settings_data.ts

```diff
@@ -90,7 +90,7 @@ export function user_can_create_multiuse_invite(): boolean {
     );
 }
 
-export function user_can_subscribe_other_users(): boolean {
+export function has_realm_wide_permission_to_subscribe_other_users(): boolean {
```

Comment:
This can also be renamed like the function used in backend.

Reply:
[Fixed](https://github.com/zulip/zulip/compare/a61a1596b3b9cafe95f41bb4cfac8085c7fda879..f58b29936f302120e3c638312bf02872eb3d4cd1)